lib/src/tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import DxcTooltip from "./Tooltip";
+import ExampleContainer from "../../.storybook/components/ExampleContainer";
+import DxcButton from "../button/Button";
+import { userEvent, within } from "@storybook/test";
+import DxcInset from "../inset/Inset";
+import Title from "../../.storybook/components/Title";
+import DxcFlex from "../flex/Flex";
+
+export default {
+  title: "Tooltip",
+  component: DxcTooltip,
+};
+
+const Tooltip = () => (
+  <>
+    <Title title="Default tooltip" theme="light" level={4} />
+    <ExampleContainer>
+      <DxcInset bottom="3rem">
+        <DxcTooltip label="Tooltip Test">
+          <DxcButton label="Hoverable button" />
+        </DxcTooltip>
+      </DxcInset>
+    </ExampleContainer>
+  </>
+);
+
+const LargeTextWithinTooltip = () => (
+  <>
+    <Title title="Default tooltip" theme="light" level={4} />
```

Comment:
Change the title and the snapshot gets cut.
![image](https://github.com/dxc-technology/halstack-react/assets/12238686/6313bb11-f2d3-4603-b16f-b30e1cfc9348)